extensions/amp-render/validator-amp-render.protoascii

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
Let's not put the validations in this PR -- this should be a separate PR when we're closer to launch.